js/src/forum/index.js

```diff
@@ -73,16 +64,15 @@ extend(DiscussionPage.prototype, 'sidebarItems', function(items) {
   const count = this.discussion.replyCount();
 
   items.add('replies', <h3>
-    <a href={app.route.discussion(this.discussion).replace('/embed', '/d')} config={m.route}>
+    <a route={app.route.discussion(this.discussion).replace('/embed', '/d')}>
       {count} comment{count == 1 ? '' : 's'}
     </a>
   </h3>, 100);
 
-  const props = items.get('controls').props;
-  props.className = props.className.replace('App-primaryControl', '');
+  const attrs = items.get('controls').attrs;
+  attrs.className = attrs.className.replace('App-primaryControl', '');
 });
 
-delete app.routes['index.filter'];
```

Comment:
To clarify for reviewers, this was removed in core; following introduces it's own route instead.